platform/broadcom/sonic-platform-modules-cel/silverstone/modules/i2c-ocores.h

```diff
@@ -0,0 +1,22 @@
+/* SPDX-License-Identifier: GPL-2.0 */
```

Comment:
kernel existing code.

Reply:
As we keep the own maintain version of i2c-cores, this file is needed for module compile.

Reply:
This will be removed after the https://github.com/Azure/sonic-linux-kernel/pull/135 is merged.

Reply:
The driver is updated to use kernel's i2c-ocores module.